src/main/java/io/jenkins/plugins/folderauth/misc/PermissionFinder.java

```diff
@@ -0,0 +1,60 @@
+package io.jenkins.plugins.folderauth.misc;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+import hudson.security.Permission;
+import hudson.security.PermissionGroup;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+/**
+ * Implements lookup for {@link Permission}s.
+ */
+// Imported from https://github.com/jenkinsci/configuration-as-code-plugin/blob/727c976d137461f146b301f302d1552ca81de75e/plugin/src/main/java/io/jenkins/plugins/casc/util/PermissionFinder.java
```

Comment:
Could we make it a public API there?

Reply:
It gone and removed 😆

Reply:
Also Matrix auth and Role auth both have them locally

Reply:
We could add it back make it public and hope that both role, folder and matrix will accept the pull request 😆

Reply:
Yeah, I think 3 copy-pastes definitely justify having an API method in JCasC or in the core